mne/realtime/epochs.py

```diff
@@ -326,16 +327,24 @@ def _process_raw_buffer(self, raw_buffer):
 
         # detect events
         data = np.abs(raw_buffer[self._stim_picks]).astype(np.int)
-        data = np.atleast_2d(data)
-        buff_events = _find_events(data, self._first_samp, verbose=verbose,
-                                   **self._find_events_kwargs)
-
+        #if there is a previous buffer check the last samples from it too        
+        if self._last_buffer != None:        
+            prev_data = np.abs(self._last_buffer[self._stim_picks,-prev_samp:])
+            prev_data = prev_data.astype(np.int)
+            data = np.concatenate((prev_data, data),axis=1)        
```

Comment:
you will need to install a pep8 checker. If you're using sublime, you can install the package `anaconda`. On spyder, you have an option to enable pyflake and pep8 checking.